jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -719,3 +720,24 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_jbpm_cluster(){
+    if [ "${KIE_SERVER_JBPM_CLUSTER^^}" = "TRUE" ]; then
+      configure_jbpm_cache
+      log_info "KIE_SERVER_JBPM_CLUSTER enabled"
```

Comment:
PLease elaborate more these log messages, what exactly is being added?
what it will configure? that's a few question that I, as user, would ask when reading these messages.

Reply:
ok